Set go_package in serialization.proto; newer Go protobuf requires it.

Chromium commit 8601ac0dd0a59. Upstream in 4f94d24.

diff --git a/src/serialization.proto b/src/serialization.proto
index b7fefcf191893..57c75a9027e6b 100644
--- a/src/serialization.proto
+++ b/src/serialization.proto
@@ -19,6 +19,8 @@ option optimize_for = LITE_RUNTIME;
 
 package rlwe;
 
+option go_package = "github.com/google/shell-encryption";
+
 // NTT Polynomial
 message SerializedNttPolynomial {
   // Coefficients of the polynomial
